internal/reconcile/rollback.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2022 The Flux authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reconcile
+
+import (
+	"context"
+	"fmt"
+
+	helmrelease "helm.sh/helm/v3/pkg/release"
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"github.com/fluxcd/pkg/runtime/conditions"
+	"github.com/fluxcd/pkg/runtime/logger"
+
+	helmv2 "github.com/fluxcd/helm-controller/api/v2beta2"
+	"github.com/fluxcd/helm-controller/internal/action"
+	"github.com/fluxcd/helm-controller/internal/release"
+	"github.com/fluxcd/helm-controller/internal/storage"
+)
+
+type Rollback struct {
+	configFactory *action.ConfigFactory
+}
+
+func (r *Rollback) Name() string {
+	return "rollback"
+}
+
+func (r *Rollback) Type() ReconcilerType {
+	return ReconcilerTypeRemediate
+}
+
+func (r *Rollback) Reconcile(ctx context.Context, req *Request) error {
```

Comment:
Missing comment.

Reply:
Yes, not planning to add them at this stage.